qa/tasks/cephfs/test_volumes.py

```diff
@@ -215,6 +216,44 @@ def test_default_uid_gid_subvolume(self):
         # remove subvolume
         self._fs_cmd("subvolume", "rm", self.volname, subvolume)
 
+    def test_subvolume_ls(self):
+        # tests the 'fs subvolume ls' command
+
+        subvolumes = []
+
+        # create subvolume1 and make sure it exists
```

Comment:
You can simplify this by,
```
for i in range(3):
    subvolumes.append(self._generate_random_subvolume_name())
    self._fs_cmd("subvolume", "create", self.volname, subvolumes[i])
    subvolpath = self._fs_cmd("subvolume", "getpath", self.volname, subvolumes[i])
    self.assertNotEqual(subvolpath, None)
```

Reply:
@ajarr We could do it in a loop, but isn't this change the order of execution?
In my code I was doing it in the following order:
1. create the subvolume
2. verify if subvolume created/exists.
3. if it exists then add to the `subvolumes` list.

Reply:
It's OK to change the order of execution.

Reply:
Also, no need to check whether the subvolume path exists after creation. It's fine to list and check. You don't check whether snapshot path exists after creation. So let's do the same here.

Reply:
Problem solved. Don't use i.
```
        for i in range(3):                                                                                                                                                                                         
            subvolname = self._generate_random_subvolume_name()                                                                                                                                                    
            self._fs_cmd("subvolume", "create", self.volname, subvolname)                                                                                                                                          
            subvolumes.append(subvolname)  
```